src/caselawclient/models/documents/body.py

```diff
@@ -51,9 +56,45 @@ def court(self) -> str:
     def jurisdiction(self) -> str:
         return self.get_xpath_match_string("/akn:akomaNtoso/akn:*/akn:meta/akn:proprietary/uk:jurisdiction/text()")
 
+    @cached_property
+    def categories(self) -> List[DocumentCategory]:
```

Comment:
I think you can just write this as `list[DocumentCategory]` and save yourself the import -- surprised the typechecking didn't change this for you.